FHEM/14_SD_UT.pm

```diff
@@ -938,15 +938,15 @@ sub SD_UT_Parse($$) {
 		### Manax MX-RCS250 [P90] ###
 		if (!$def && $protocol == 90) {
 			$deviceCode = substr($rawData,0,4);
-			$devicedef = "Manax " . $deviceCode;
-			$def = $modules{SD_UT}{defptr}{$devicedef};
+			$devicedef = "Manax " . $deviceCode if (!$def);
+			$def = $modules{SD_UT}{defptr}{$devicedef} if (!$def);
 		}
 
 		### ESTO KL_RF01 [P93] ###
 		if (!$def && $protocol == 93 || $protocol == 90) {
 			$deviceCode = substr($rawData,0,4);
-			$devicedef = "KL_RF01 " . $deviceCode;
-			$def = $modules{SD_UT}{defptr}{$devicedef};
+			$devicedef = "KL_RF01 " . $deviceCode if (!$def);
```

Comment:
Wieso änderst Du nich die Zeile 946 ab in

`		if (!$def && ($protocol == 93 || $protocol == 90)) {`

Mir scheint das sonst irgendwie halb doppelt gemoppelt zu sein :)

Reply:
Ja man kann es auch so schreiben. Wenn du es wünscht, passe ich es noch an :-)

Reply:
Sonst wird gefühlt 4x auf !def geprüft, was ja eigentlich bereits in der Zeile 946 erfolgen sollte.
Ob Protokoll == 90 an dieser Stelle noch sinnvoll ist, da dies bereits ein paar Zeilen oben drüber geprüft wird, habe ich noch nicht bewertet.